src/tasks/minions/HunterActivity/birdhouseActivity.ts

```diff
@@ -56,6 +65,26 @@ export default class extends Task {
 				str = `${user}, ${user.minionName} finished collecting 4x full ${birdhouseToCollect.name}.`;
 			}
 
+			for (let i = 0; i < 4; i++) {
```

Comment:
This should probably be a function though. 

Is there already a function for skilling clues? I know this isn't quite the same, but as long as we pass in the appropriate clue weightings and roll() chance it should be okay.  Just a thought, I don't think it's required here